spec/index.md

```diff
@@ -2470,6 +2470,50 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `take_canister_snapshot` {#ic-take_canister_snapshot}
+
+This method takes a snapshot of the specified canister. A snapshot consists of the wasm memory, stable memory, certified variables, wasm chunk store and wasm binary.
+
+Subsequent `take_canister_snapshot` calls will create a new snapshot. However, a `take_canister_snapshot` call might fail if the maximum number of snapshots per canister is reached. This error can be avoided by providing a snapshot ID via the optional `replace_snapshot` parameter. The snapshot identified by the specified ID will be deleted once a new snapshot has been successfully created. Currently, only one snapshot per canister is allowed.
```

Comment:
The spec says I can only take one snapshot per canister, but I tried the local replica, and I can take more than 1 snapshots. My local replica is configured as system subnet, not sure if that makes a difference.

Reply:
Not sure this is true anymore, it depends which replica version you used. The latest should only allow 1 snapshot to be present at a time. You have to call `take_canister_snapshot` with `replace_snapshot_id` to succeed (which will overwrite the previous snapshot).